modules/metastore/src/ValidMetadataFactory.php

```diff
@@ -3,13 +3,15 @@
 namespace Drupal\metastore;
 
 use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\metastore\Reference\HelperTrait;
 use RootedData\RootedJsonData;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * Service.
  */
 class ValidMetadataFactory implements ContainerInjectionInterface {
+  use HelperTrait;
```

Comment:
@klimp-drupal I don't think it makes sense to pull in a trait from the Reference namespace, which is unrelated to ValidMetadataFactory. Also all we're getting from that trait is a reference to the Uuid class, which we could invoke directly. So just use Service\Uuid and then call something like `(new Uuid)->generate($schema_id, $json_string);`

Reply:
Fixed